### ISSUE TITLE
Bots not moving in HL campaign.

### ISSUE THREAD

**Issue by mrwallace888** [rcbotCheeseh/RCBotSven5#15]
They don't do anything on the HL campaign no matter what map/chapter I try, even though this comes with waypoints. Other maps are fine.

**Comment by caxanga334** [comment]
Looking at the repository files, no waypoints are included for most of the new Half-Life 1 campaign maps (Sven 5 or newer). Only the original HL1 campaign maps (Sven 4.8 or older) have waypoints.

**Comment by mrwallace888** [comment]
There's new HL1 campaign maps?

**Comment by caxanga334** [comment]
Yes. Sven 4.8 or older hl1 campaign maps are **c#a#** where # is a number. The new campaign maps in Sven 5 are named **hl_c#**.
It's still the same maps but in Sven 5 the hl1 campaign was merged to reduce the number of map files.
